lib/ui/screens/boxes_screen.dart

```diff
@@ -1,12 +1,18 @@
 import 'package:flutter/material.dart';
+import 'package:get_it/get_it.dart';
 import 'package:sliver_tools/sliver_tools.dart';
 import 'package:zachranobed/common/constants.dart';
 import 'package:zachranobed/common/helper_service.dart';
 import 'package:zachranobed/extensions/build_context_extensions.dart';
 import 'package:zachranobed/models/box_movement_list_info.dart';
+import 'package:zachranobed/services/box_movement_service.dart';
 import 'package:zachranobed/ui/widgets/box_movement_list.dart';
 import 'package:zachranobed/ui/widgets/button.dart';
 
+import '../widgets/empty_page.dart';
+import '../widgets/error_page.dart';
+import '../widgets/loading_page.dart';
```

Comment:
Nit: Pls change to absolute imports 😇 

Also in `lib/ui/screens/donations_screen.dart` and `lib/ui/widgets/empty_page.dart`

Reply:
Fixed